src/content/digest/letter-from-the-guest-editor.md

```diff
@@ -1,47 +1,41 @@
 ---
 title: Letter from the Guest Editor
-created-on: 2024-08-26T15:25:00.000Z
-updated-on: 2024-08-26T15:25:00.000Z
-published-on: 2024-08-26T15:25:00.000Z
-description: "Welcome to the inaugural edition of The Filecoin Ecosystem Digest:
-  All Systems Go - a journey into the transformative power of the decentralized
-  web and its far-reaching implications across various fields. "
+created-on: 2024-08-27T10:15:00.000Z
+updated-on: 2024-08-27T10:15:00.000Z
+published-on: 2024-08-27T10:15:00.000Z
+description: Letter from the Guest Editor
 issue-number: 1
-article-number: 2
-publication-date: 2024-08-26T15:26:00.000Z
-image:
-  src: /assets/images/01-lockheed-p-1080.png
+article-number: 1
+publication-date: 2024-08-27T10:15:00.000Z
```

Comment:
What is this for? Where in the design it used?

Reply:
From conversations I understood with Gary and Filipa, we just want to have the issue date.